src/main/java/com/dell/isg/smi/service/server/inventory/controller/ServerInventoryController.java

```diff
@@ -83,9 +84,7 @@ public ServerHardwareInventory inventory(@RequestBody Credential payload) {
 
 
     @RequestMapping(value = "/summary", method = RequestMethod.POST, headers = "Accept=application/json", consumes = "application/json", produces = "application/json")
-    @ApiOperation(value = "/summary", nickname = "system", notes = "This operation allow user to get partial server information throu wsman.", response = HwSystem.class)
-    // @ApiImplicitParams({
-    // @ApiImplicitParam(name = "credential", value = "Credential", required = true, dataType = "Credential.class", paramType = "Body", defaultValue = "no default") })
+    @ApiOperation(value = "/summary", nickname = "summary", notes = "This operation allow user to get server system information (System View) using wsman protocol.", response = HwSystem.class)
```

Comment:
This operation allows the user to retrieve server system information (System View) using the WSMan protocol.